requirements.txt

```diff
@@ -10,6 +10,7 @@ awesomeversion==22.6.0
 bcrypt==3.1.7
 certifi>=2021.5.30
 ciso8601==2.2.0
+fnvhash==0.1.0
```

Comment:
This is a dependency of `homekit` and `recorder`, and not a core dependency?
I don't see any part of core using this. Could you elaborate on this addition?

Reply:
It looks like if you don't have this installed, Home Assistant immediately quits on startup _without_ an error about not having it. I found this PR while trying to resolve #76677 in a fresh dev container.

Reply:
Right, but this sounds like a workaround in that case? The question is: why doesn't the core ensure those dependencies before setting those up?

Reply:
From my personal testing, and to hopefully assist with investigation. 
It appears if installing from WSL2 and Ubuntu 20.04 with Python 3.8 (default WSL2 distro) it will startup correctly. 
However it fails in 22.04 with Python 3.10.4 without those dependencies installed.

Reply:
For me it would be perfectly okay if this was added as deps of homekit and recorder, **provided that I can start a fresh hass after script/setup**.

Reply:
That is not a solution? See <https://github.com/home-assistant/core/pull/76597#discussion_r945333567>

Reply:
I understand the deps need to go onto homekit and recorder (probably already there), but whatever is causing early initialization to bomb out before installing those homekit / recorder deps (likely recorder, that was on my logs) needs to be fixed.